ISSUES.md

```diff
@@ -10,6 +10,20 @@ Detailed information about each Bundler command, including help with common prob
 
 ## Troubleshooting
 
+### Permission denied when installing bundler
+Certain operating systems such as MacOS and Ubuntu have versions of Ruby that requires evelated privileges to install gems.
+
+```
+ERROR:  While executing gem ... (Gem::FilePermissionError)
+    You don't have write permissions for the /Library/Ruby/Gems/2.0.0 directory.
+```
+
+This is typically solved by prepending the command with `sudo`.
```

Comment:
The issue with `--user-install` is that it means you have to also add the user install `bin` directory to your $PATH. If you add that, definitely call this out.